daml/Marketplace/Rule/AllocationAccount.daml

```diff
@@ -40,14 +40,15 @@ template AllocationAccountRule
       creditAccount : Account -> Asset -> Update (ContractId AssetDeposit)
       creditAccount = \account asset -> exerciseByKey @AssetSettlementRule account.id AssetSettlement_Credit with ctrl = nominee, ..
 
-    controller nominee, account.owner can
-      nonconsuming Transfer : ContractId AssetDeposit
+    nonconsuming choice Transfer : ContractId AssetDeposit
         with
           transferTo : Account
           depositCid : ContractId AssetDeposit
+        controller nominee, account.owner, transferTo.owner
```

Comment:
Have you tested that the trading and the auction settlements are still both working after adding `transferTo.owner` to this controller ?

Reply:
Yes